core-libs/setup/ssr/logger/loggers/express-server-logger.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Request } from 'express';
+import {
+  DefaultExpressServerLogger,
+  ExpressServerLoggerContext,
+} from './express-server-logger';
+
+describe('ExpressServerLogger', () => {
```

Comment:
[MINOR]

It's a test suite for `DefaultExpressServerLogger`. Let's adjust the description and the file name.
```suggestion
describe('DefaultExpressServerLogger', () => {
```